src/rotp/lang/fr/labels.txt

```diff
@@ -521,18 +510,64 @@ GAME_SETTINGS_AUTOBOMBARD_NEVER   | Jamais
 GAME_SETTINGS_AUTOBOMBARD_YES     | Toujours
 GAME_SETTINGS_AUTOBOMBARD_WAR     | Seulement en guerre
 GAME_SETTINGS_AUTOBOMBARD_INVADE  | Seulement en guerre et pas invasion
+GAME_SETTINGS_AUTOBOMBARD_NO_HELP     | Pas d'automatisation ! Toujours demander des commandes fraîches!
+GAME_SETTINGS_AUTOBOMBARD_NEVER_HELP  | Pas d'automatisation ! Toujours demander des commandes fraîches!
```

Comment:
Ce texte en double existe aussi en Anglais. Bug?

Reply:
Yes Bug! Should be:
GAME_SETTINGS_AUTOBOMBARD_NEVER_HELP  | No questions asked! Never bomb!